geo_to_hca/utils/get_attribs.py

```diff
@@ -138,35 +143,36 @@ def get_attributes_library_protocol(experiment_package: object) -> []:
             instrument = ''
     return [experiment_id,library_construction_protocol,instrument]
 
+
 def get_attributes_bioproject(xml_content: object, bioproject_accession: str) -> [str,str,str,str]:
     bioproject_metadata = xml_content.find('DocumentSummary')
     try:
         project_name = bioproject_metadata.find("Project").find('ProjectDescr').find('Name').text
     except:
-        print("no project name")
+        log.info("no project name")
         project_name = None
     try:
         project_title = bioproject_metadata.find("Project").find('ProjectDescr').find('Title').text
     except:
-        print("no project title")
+        log.info("no project title")
         project_title = None
     try:
         project_description = bioproject_metadata.find("Project").find('ProjectDescr').find('Description').text
     except:
         project_description = ''
-        print("no project description")
+        log.info("no project description")
     project_publication = bioproject_metadata.find("Project").find('ProjectDescr').find('Publication')
     try:
         if project_publication.find('DbType').text == 'Pubmed' or project_publication.find('DbType').text == 'ePubmed':
             project_pubmed_id = project_publication.find('Reference').text
     except:
-        print("No publication for project %s was found: searching project title in EuropePMC" % (bioproject_accession))
+        log.info("No publication for project %s was found: searching project title in EuropePMC" % (bioproject_accession))
     if not project_publication or not project_pubmed_id:
```

Comment:
I can see that you only replaced `print` to logging in these lines, but this conditional statements are going very deep. Would be nice to refactor it to a cleaner code.

Reply:
This script is a big mess that needs a lot of cleaning.
The task at hand is about integrating it to the UI and broker as a "black box", so although I totally agree with your comment @ke4, I think it could be done at a later stage.